### ISSUE TITLE
11.2i crashs on linux dedicated server

### ISSUE THREAD

**Issue by B0bbyD0llar** [MatrexsVigil/harvestcraft#287]
hi,

on my dedicated linux server i have some trouble with 11.2i. 

i have tried 2 versions of forge. here my crash-logs:

https://gist.github.com/B0bbyD0llar/0dc6e845d4eea5afec7adcfae937c911

and

https://gist.github.com/B0bbyD0llar/e32a1649b03b041611f06f931e0271eb

Any idea?

PS: on my windows client all works fine.

**Comment by An-Sar** [comment]
Same issue here. 

TileEntitySpecialRenderer is client side only. Expect any dedicated server to crash, nothing specific to linux.

**Comment by kreezxil** [comment]
confirmed:

 1.11.2i crashes dedicated server Forge 1.11.2 build 2311

Notes:
     on startup invalid side SERVER

crash report:
https://gist.github.com/ca85cc43be41bb6dc6d5866113cd1d9f

fml server latest:
https://gist.github.com/0f1c8ad271175321a77e0e0491c22b23

**Comment by samip5** [comment]
Confirmed, 1.11.2i crashes dedicated server Forge 1.11.2 build 2310

Missing ClientSide annotation as it's for client only.

Logs:
https://gist.github.com/samip5/93356059d79040969349d2ae58f784b6

**Comment by Wizardon777** [comment]
Also on windows server, crash report included.
[crash-2017-06-03_18.29.30-server.txt](https://github.com/MatrexsVigil/harvestcraft/files/1049680/crash-2017-06-03_18.29.30-server.txt)

**Comment by king05** [comment]
i got the same crash on startup with newest forge 2312
crash report: https://pastebin.com/hfFs4667
lastest log: https://pastebin.com/WEGq1Vi0

**Comment by B0bbyD0llar** [comment]
it doesnt mater which forge version. No version works!

**Comment by samip5** [comment]
@B0bbyD0llar That's true, but the error is caused by an client side only code which tries to get loaded on the server, but it's unable to find it as it's not present on that side.

**Comment by B0bbyD0llar** [comment]
@samip5 Ahh OK interesting. 
What line do you recognize?

**Comment by samip5** [comment]
@B0bbyD0llar 
```java
This line: net/minecraft/client/renderer/tileentity/TileEntitySpecialRenderer 
```
Is only for client as you can see from the class name. 

This line: 
```java
java.lang.RuntimeException: Attempted to load class bsl for invalid side SERVER
```
says exactly what the problem is.

**Comment by B0bbyD0llar** [comment]
@samip5 
thx dude. So I've learned something again.

**Comment by EmosewaGamer** [comment]
It's not only on Linux, people on Windows also get it.
